signature-aggregator/README.md

```diff
@@ -0,0 +1,64 @@
+# Signature Aggregator
+
+This directory contains a lightweight stand-alone API for requesting signatures for a Warp message from subnet validators.
+It is also used by the relayer for gathering signatures when configured to use AppRequest instead of the WarpSignature RPC client.
+
+## Building
+
+To build the application run `scripts/build_signature_aggregator.sh` which will output the binary to `signature-aggregator/build/signature-aggregator` path by default.
+
+## Running
+
+To run the binary you must supply a config file via `./signature-aggregator --config-file`
+Currently required configurations are a small subset of the [relayer configuration](https://github.com/ava-labs/awm-relayer?tab=readme-ov-file#configuration).
+
+Namely:
+- `LogLevel`: string
+- `PChainAPI`: APIConfig
+- `InfoAPI` : APIConfig
+- `APIPort` : (optional) defaults to 8080
+
+Sample config that can be used for local testing is `signature-aggregator/sample-signature-aggregator-config.json`
+
+## Interface
+
+The only exposed endpoint is `/aggregate-signatures/by-raw-message`  expecting `application/json` encoded request with the following body. Note that all the fields are optional but at least one of `data` or `justification` must be non-empty:
+```json
+{
+    "data": "",               // (string) hex-encoded unsigned message bytes to be signed
+    "justification": "",      // (string) hex-encoded bytes to supply to the validators as justification
+ .  "signing-subnet-id": "",  // (string) hex or cb58 encoded signing subnet ID. Defaults to source blockchain's subnet from data if omitted.
+    "quorum-num": 67          // (int) quorum percentage required to sign the message. Defaults to 67 if omitted
+}
+```
+
+## Sample workflow
```

Comment:
Can you add a note here that this example workflow will only work for recent messages due to validator churn?